src/main/java/uk/gov/hmcts/divorce/divorcecase/model/FinalOrder.java

```diff
@@ -89,7 +89,7 @@ public enum Granted implements HasLabel {
         label = "Has applicant1 applied for a final order?",
         access = {DefaultAccess.class}
     )
-    private YesOrNo applicant1AppliedForFinalOrderFirst;
+    private YesOrNo applicant1AppliedForFinalOrder;
```

Comment:
you won't be able to rename fields once they are added. You will need to use Retired field to do the migration